momento-sdk/src/main/java/momento/sdk/responses/storage/GetResponseFound.java

```diff
@@ -0,0 +1,5 @@
+package momento.sdk.responses.storage;
+
+public interface GetResponseFound {
+  StorageValue value();
+}
```

Comment:
Is there value in this extra type or should the "found" case simply yield a StorageValue, which is the only thing a GetResponse should yield in the found case?

Reply:
I had considered this as well. For the first cut I have kept the response class in parallel with the wire response (ie a `GetResponse` has a `value` which is `oneof` ...). This made more sense when we had `Success` vs `Error` and the `Success` was a found vs not found. That is no longer the case.

Right now there is no other data in the response message. There is no danger in rolling up the value accessors here as long as we avoid name conflicts in the future.